runtime/cudaq/builder/kernel_builder.cpp

```diff
@@ -33,6 +33,8 @@
 
 #include <numeric>
 
```

Comment:
nit: why the blank lines?

Reply:
iostream is just here for some cout's I added to debug. I just separated it from the other includes so it doesn't get lost in the sauce 😃 I'll be reverting things back shortly

Reply:
https://www.youtube.com/watch?v=yU2GqcjJcdM